ingest/src/main/java/com/microsoft/azure/kusto/ingest/ManagedStreamingIngestClient.java

```diff
@@ -206,7 +210,6 @@
     /**
      * {@inheritDoc}
      * <p>
-     * This method behaves differently from the rest for {@link ManagedStreamingIngestClient} - since a blob already exists it makes more sense to enqueue it rather than downloading and streaming it, thus ManagedStreamingIngestClient skips the streaming retries and sends it directly to the queued client.</p>
      */
     @Override
     public IngestionResult ingestFromBlob(BlobSourceInfo blobSourceInfo, IngestionProperties ingestionProperties)
```

Comment:
Again - this is a long & complex method that shares a lot of the code with ingest from stream.

Try to combine & resuse as much as possible

Reply:
Agreed with you there - disagree here . Not much in common